x/wasm/module_test.go

```diff
@@ -238,12 +238,12 @@ func TestHandleExecute(t *testing.T) {
 	require.Equal(t, "cosmos14hj2tavq8fpesdwxxcu44rty3hh90vhuc53mp6", contractBech32Addr)
 	// this should be standard x/wasm message event,  init event, plus a bank send event (2), with no custom contract events
 	require.Equal(t, 4, len(res.Events), prettyEvents(res.Events))
-	require.Equal(t, "transfer", res.Events[0].Type)
-	require.Equal(t, "wasm", res.Events[1].Type)
-	assertAttribute(t, "_contract_address", contractBech32Addr, res.Events[1].Attributes[0])
+	require.Equal(t, "message", res.Events[0].Type)
```

Comment:
This is very nice and clear...

message (top-level), transfer (pay fees sent with message), instantiate + wasm (actual contract execution)